Sources/Logging/Strings/OfferingStrings.swift

```diff
@@ -38,6 +38,8 @@ enum OfferingStrings {
     case configuration_error_no_products_for_offering
     case offering_empty(offeringIdentifier: String)
     case product_details_empty_title(productIdentifier: String)
+    case unknown_package_type(Package)
+    case custom_package_type(Package)
```

Comment:
I see the cases above have a parameter name in addition to the type. Is that ok? And if so, it might be a good idea to consolidate them.

Reply:
It's useful for parameters like `String`s because it's not clear what they contain. But a package is always going to be a package 😅